src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -266,7 +266,9 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
           <Spacer y={4} />
           <Flex width="100%" justifyContent="space-between">
             <Button
-              onClick={() => {
+              // prevent a parent formik context from treating this as a submit button
+              type="button"
+              onClick={e => {
```

Comment:
Is this comment accurate? Without `e.preventDefault()` it should still behave as a button